### ISSUE TITLE
desktop app ?

### ISSUE THREAD

**Issue by jkanche** [huytd/pomoday-v2#36]
I was wondering if we can build this into a desktop app using electron ?

**Comment by huytd** [comment]
Yes, It's doable but I think it's not a priority for now. I do use the desktop version with the help of [nativefier](https://github.com/jiahaog/nativefier).

**Comment by oh-frontend1** [comment]
Maybe we just have to apply PWA (Progressive Web App) to build an offline-first web app, then we can install it to desktop/mobile using Chrome (https://www.howtogeek.com/fyi/how-to-install-progressive-web-apps-pwas-in-chrome/)

**Comment by huytd** [comment]
Sure. Offline support is already planned in the next 2 version.

**Comment by deepDiverPaul** [comment]
Using `nativefier` worked for me:

`$ npm install nativefier -g`
`$ nativefier "https://pomoday.app" --name "Pomoday"`

The app-icon depends on your OS. See [nativefier api](https://github.com/jiahaog/nativefier/blob/master/docs/api.md#icon)

**Comment by huytd** [comment]
Apparently Nativefier has the ability to set the icon yourself, using `--icon=file-name.png` or something.

Here's what I'm using on my machine right now :D 
<img width="346" alt="image" src="https://user-images.githubusercontent.com/613943/70257333-eecc5080-173e-11ea-9b55-1d2a82936e8b.png">

If anyone interested, here's the icons you can use:

![pomoday-icon-local](https://user-images.githubusercontent.com/613943/70257360-fab81280-173e-11ea-9dff-cbf41346ca79.png)
![pomoday-icon-dev](https://user-images.githubusercontent.com/613943/70257362-fab81280-173e-11ea-9b30-8d5d1c36bf95.png)
![pomoday-icon](https://user-images.githubusercontent.com/613943/70257363-fab81280-173e-11ea-85b9-80a347644bcf.png)
